resources/sql/queries.sql

```diff
@@ -207,6 +207,36 @@ WHERE 1=1
 /*~ ) ~*/
 ;
 
+-- :name get-form-template :? :1
+SELECT
+  id,
+  organization,
+  title,
+  start,
+  endt as "end",
+  fields::TEXT,
+  enabled,
+  archived,
+  TRUE AS active -- TODO implement active and archiving
```

Comment:
`rems.db.core/assoc-active` can be used to calculate active.

Reply:
Hm, I copy pasted from other queries. I have to see if I will fix the other parts here or in other PR.